index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
+      code</a> <a>invalid argument</a>.
+
+      <li><p>If <var>range</var> is a <a>Number</a>:
+        <ol>
+          <li>If <var>range</var> is not an integer or is less than 0,
+      return <a>error</a> with <a>error code</a> <a>invalid
+      argument</a>
+
+          <li>Append <var>range</var> to <var>pages</var>
+        </ol>
+      <p>Otherwise:
+        <ol>
+          <li>Let <var>rangeParts</var> be the result of
+          splitting <var>range</var> on a "<code>-</code>" character.
+
+          <li>If <var>rangeParts</var> has fewer than 1 or more than 2
+          elements, return <a>error</a> with <a>error code</a> <a>invalid
+          argument</a>.
+
+          <li>If rangeParts has one element, append the result
+          of <a>trying</a> to <a>parse as an integer</a> the first
+          element of <var>rangeParts</var>
+          to <var>pages</var>.
+            <p>Otherwise:
+              <ol>
+                <li>If the first element of <var>rangeParts</var>
+                  is <a>equivalent to an empty string</a>,
+                  let <var>lowerBound</var>
+                  be <code>1</code>. Otherwise
+                  let <var>lowerBound</var> be the result
+                  of <a>trying</a> to <a>parse as an integer</a> the
+                  first element of <var>rangeParts</var>.
+
+                <li>If the second element of <var>rangeParts</var>
+                  is <a>equivalent to an empty string</a>
+                  let <var>upperBound</var>
+                  be <var>totalPages</var>. Otherwise
+                  let <var>upperBound</var> be the result
+                  of <a>trying</a> to <a>parse as an integer</a> the
+                  second element of <var>rangeParts</var>.
+
+                <li>If <var>lowerBound</var> is greater
+                than <var>upperBound</var>, return <a>error</a>
+                with <a>error code</a> <a>invalid argument</a>.
+
+                <li>Append all integers in the inclusive
+                range <var>lowerBound</var> to <var>upperBound</var>
+                to <var>pages</var>
+              </ol>
+        </ol>
+   <li>Return <a>success</a> with data <var>pages</var>.
+</ol>
+
+<p>A <a>String</a> is <dfn>equivalent to an empty string</dfn> if it
+has zero length after removing all <a>whitespace</a> characters.
+
+<p>When required to <dfn>parse as an integer</dfn> with
+argument <var>input</var> an implementation must:
+
+<ol>
+  <li>Let <var>stripped</var> be the result of stripping all leading
+  and trailing <a>whitespace</a> characters from <var>input</var>.
+
+  <li>If <var>stripped</var> has zero length, return <a>error</a> with
+  status <a>invalid argument</a>.
+
+  <li>If <var>stripped</var> contains any characters outside the
+  range <code>U+0030</code> - <code>U+0039</code> (i.e. 0 - 9)
+  inclusive, return <a>error</a> with status <a>invalid argument</a>.
+
+  <li>Let <var>output</var> be the result of calling <a>parseInt</a>
```

Comment:
Trying to call? What happens if `parseInt` is not successful?

Reply:
The previous steps ensure we have a string containing only characters in the range 0-9. I think in that case `parseInt` has to succeed.